File: ResourceManager/Models/Configurations/PasswordConfiguration.cs
using Microsoft.EntityFrameworkCore;
using Microsoft.EntityFrameworkCore.Metadata.Builders;

namespace ResourceManager.Models.Configurations
{
    public class PasswordConfiguration : IEntityTypeConfiguration<Password>
    {
        public void Configure(EntityTypeBuilder<Password> builder)
        {
            builder.ToTable("Passwords");
            builder.HasKey(x => x.ID);

            builder.Property(x => x.ID).ValueGeneratedOnAdd();
            builder.Property(x => x.CreatedDate).HasDefaultValueSql("date('now','localtime')");
            builder.Property(x => x.ModifiedDate).HasDefaultValueSql("date('now','localtime')");
            builder.Property(x => x.Name).HasColumnType("nvarchar(100)").IsRequired();
            builder.Property(x => x.Username).HasColumnType("nvarchar(50)").IsRequired();
            builder.Property(x => x.Pass).HasColumnType("nvarchar(100)").IsRequired();
            builder.Property(x => x.Url).HasColumnType("nvarchar(100)").IsRequired(false);
            builder.Property(x => x.Description).HasColumnType("nvarchar(255)").IsRequired(false);
            builder.HasIndex(x => x.Name).IsUnique();

            builder.HasOne(x => x.User)
                .WithMany(x => x.Passwords)
                .HasForeignKey(x => x.UserID);
                
        }
    }
}

File: ResourceManager/Models/Configurations/PasswordConfiguration.vb
Imports Microsoft.EntityFrameworkCore
Imports Microsoft.EntityFrameworkCore.Metadata.Builders

Namespace ResourceManager.Models.Configurations
    Public Class PasswordConfiguration
        Implements IEntityTypeConfiguration(Of Password)
        Public Sub Configure(ByVal builder As EntityTypeBuilder(Of Password)) Implements IEntityTypeConfiguration(Of Password).Configure
            builder.ToTable("Passwords")
            builder.HasKey(Function(x) x.ID)

            builder.Property(Function(x) x.ID).ValueGeneratedOnAdd()
            builder.[Property](Function(x) x.CreatedDate).HasDefaultValueSql("date('now','localtime')")
            builder.[Property](Function(x) x.ModifiedDate).HasDefaultValueSql("date('now','localtime')")
            RelationalPropertyBuilderExtensions.HasColumnType(builder.Property(Function(x) x.Name), "nvarchar(100)").IsRequired()
            RelationalPropertyBuilderExtensions.HasColumnType(builder.Property(Function(x) x.Username), "nvarchar(50)").IsRequired()
            RelationalPropertyBuilderExtensions.HasColumnType(builder.Property(Function(x) x.Pass), "nvarchar(100)").IsRequired()
            RelationalPropertyBuilderExtensions.HasColumnType(builder.Property(Function(x) x.Url), "nvarchar(100)").IsRequired(False)
            RelationalPropertyBuilderExtensions.HasColumnType(builder.Property(Function(x) x.Description), "nvarchar(255)").IsRequired(False)
            builder.HasIndex(CType(Function(x) x.Name, Expressions.Expression(Of Func(Of Password, Object)))).IsUnique()

            builder.HasOne(Function(x) x.User).WithMany(Function(x) x.Passwords).HasForeignKey(Function(x) x.UserID)

        End Sub
    End Class
End Namespace
